lib/csetgrp.gi

```diff
@@ -629,6 +629,14 @@ function(d)
   Print(ViewString(d));
 end);
 
+InstallMethod(IsBiCoset,"test property",true,[IsRightCoset],0,
+function(c)
+local s,r;
+  s:=ActingDomain(c);
+  r:=Representative(c);
+  return ForAll(GeneratorsOfGroup(s),x->r*x/r in s);
```

Comment:
Perhaps test for `x^r in s` instead, as that will be a bit more efficient in some cases (e.g. for permutations)?

Reply:
Yes, if this ever could be time-critical this has the potential to be faster. I decided on the inverse-conjugation as it translated directly to the condition as I had written it doen on paper.